core/src/main/kotlin/com/getstrm/pace/processing_platforms/bigquery/BigQueryTransformer.kt

```diff
@@ -38,6 +40,29 @@ object BigQueryTransformer : ProcessingPlatformTransformer(BigQueryRenderer) {
             )
         }
 
+    override fun hash(
+        field: DataPolicy.Field,
+        hash: DataPolicy.RuleSet.FieldTransform.Transform.Hash
+    ): Field<*> {
+        val dataType = field.toJooqField().dataType
+        // If the field is a string, we use SHA256, otherwise we use FARM_FINGERPRINT.
+        return if (dataType.isNumeric) {
+            DSL.field(
+                "FARM_FINGERPRINT(CAST({0} AS STRING))",
+                Any::class.java,
+                DSL.field(DSL.unquotedName(renderName(field.fullName()))),
+            )
+        } else if (dataType.isString) {
+            DSL.field(
+                "TO_HEX(SHA256(CAST({0} AS STRING)))",
+                Any::class.java,
+                DSL.field(DSL.unquotedName(renderName(field.fullName()))),
```

Comment:
Is `DSL.field(field.fullName())` perhaps equivalent to this?

Reply:
nope thats an unquoted, not-backticked name. I need it to be `` `<FIELD_NAME>` ``.